models/articles-models.js

```diff
@@ -14,4 +14,19 @@ function selectArticleById(article_id) {
     });
 }
 
-module.exports = {selectArticleById};
+function selectArticles() {
+  return db
+    .query(
+      `
+  SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.article_id) AS comment_count FROM articles
+  FULL OUTER JOIN comments ON articles.article_id = comments.article_id
```

Comment:
LEFT JOIN may be more appropriate here as it only returns the unmatched rows from the left table (articles) whereas FULL OUTER JOIN returns unmatched rows from both tables (comments as well). It might be totally fine here because we are grouping by article_id (I've not checked!) but something I noted and think you should be aware of, should it cause unexpected errors down the line. I'd recommend just double checking the numbers you get back are what you expect for your test data 😄